catroid/src/org/catrobat/catroid/content/bricks/WhenRaspiPinChangedBrick.java

```diff
@@ -55,6 +55,10 @@ public WhenRaspiPinChangedBrick(RaspiInterruptScript script) {
 		if (script != null) {
 			pinString = script.getPin();
 			eventString = script.getEventValue();
+
+			if (script.isCommentedOut()) {
+				setCommentedOut(true);
+			}
 		}
```

Comment:
because the comment out status of a Brick is copied only when needed, e.g. when packing a Script to the Backpack. With that I don't have to modify the clone() method of each brick.